rbmc-cfam-daemon/src/meson.build

```diff
@@ -14,6 +14,7 @@ sources = [
     'cfam_fields.cpp',
```

Comment:
> The idea for use is that readAll() will be called periodically to
obtain the sibling's redundancy related values.

Is there no interrupt that ensures the sibling CFAM scratch register is updated?

Reply:
I don't think so but I will doublecheck.